recipes/lemon/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    ..
+
+cmake --build . --config Release
+
+cmake --build . --config Release --target error_test
+cmake --build . --config Release --target bfs_test
+
+ctest -R error_test
+ctest -R bfs_test
```

Comment:
Ok, that's fine. Building a couple tests by hand and running them is probably fine then.

Do you recall which test doesn't compile for future reference?